doc/releases/changelog-0.11.0.md

```diff
@@ -23,106 +26,104 @@
+    * `S` gate → :math:`P = Z` and :math:`\theta = \tfrac{\pi}{4}` 
+    * `T` gate → :math:`P = Z` and :math:`\theta = \tfrac{\pi}{8}` 
+    * `CNOT` gate → :math:`P = (Z \otimes X)(-Z \otimes \mathbb{1})(-\mathbb{1} \otimes X)` and :math:`\theta = \tfrac{\pi}{4}` 
 
-  
-  Example:
   ```python
-    @qjit(keep_intermediate=True)
-    @pipeline({"to_ppr": {}, "commute_ppr": {}})
-    @qml.qnode(qml.device("null.qubit", wires=1))
-    def circuit():
-        qml.H(0)
-        qml.T(0)
-        return measure(0)
-    ```
-  
-  The circuit program that generated from this pass is currrently not executable on any backend. For more information regarding to PPM, please refer to [(Pauli Product Measurement)](https://pennylane.ai/compilation/pauli-product-measurement)
+  import catalyst
 
-* Absorbing Clifford Pauli Product Rotation (PPR) operations into the final Pauli Product Measurement (PPM) is not availble through the :func:`~.catalyst.passes.ppr_to_ppm` pass transform. The output from this pass consists of non-Clifford PPRs and PPMs.
-  [(#1577)](https://github.com/PennyLaneAI/catalyst/pull/1577)
+  @catalyst.qjit(keep_intermediate=True)
+  @catalyst.passes.to_ppr
+  @qml.qnode(dev)
+  def circuit():
+      qml.H(0)
+      qml.S(1)
+      qml.T(0)
+      qml.CNOT([0, 1])
+      m1 = catalyst.measure(wires=0)
+      m2 = catalyst.measure(wires=1)
+      return m1, m2
+  ```
 
-  Example:
+  This circuit has the following representation in MLIR:
+  ```mlir
+    . . .
+      %0 = quantum.alloc( 2) : !quantum.reg
+      %1 = quantum.extract %0[ 1] : !quantum.reg -> !quantum.bit
+      %2 = qec.ppr ["Z"](4) %1 : !quantum.bit
+      %3 = quantum.extract %0[ 0] : !quantum.reg -> !quantum.bit
+      %4 = qec.ppr ["Z"](4) %3 : !quantum.bit
+      %5 = qec.ppr ["X"](4) %4 : !quantum.bit
+      %6 = qec.ppr ["Z"](4) %5 : !quantum.bit
+      %7 = qec.ppr ["Z"](8) %6 : !quantum.bit
+      %8:2 = qec.ppr ["Z", "X"](4) %7, %2 : !quantum.bit, !quantum.bit
+      %9 = qec.ppr ["Z"](-4) %8#0 : !quantum.bit
+      %10 = qec.ppr ["X"](-4) %8#1 : !quantum.bit
+      %mres, %out_qubits = qec.ppm ["Z"] %9 : !quantum.bit
+      %mres_0, %out_qubits_1 = qec.ppm ["Z"] %10 : !quantum.bit
+    . . .
+  ```
+
+  * :func:`catalyst.passes.commute_ppr <~.passes.commute_ppr>`: Commuting Clifford PPR operations 
+    (PPRs with :math:`\theta = \tfrac{\pi}{4}`) to the end of a circuit, past non-Clifford PPRs (PPRs 
+    with :math:`\theta = \tfrac{\pi}{8}`)
+  
   ```python
-    @qjit(keep_intermediate=True)
-    @pipeline({"to_ppr": {}, "commute_ppr": {}, "ppr_to_ppm": {}})
-    @qml.qnode(qml.device("null.qubit", wires=1))
-    def circuit():
-        qml.H(0)
-        qml.T(0)
-        return measure(0)
+  @catalyst.qjit(keep_intermediate=True)
+  @catalyst.pipeline({"to_ppr": {}, "commute_ppr": {}})
+  @qml.qnode(qml.device("null.qubit", wires=1))
+  def circuit():
+      qml.H(0)
+      qml.T(0)
+      return measure(0)
```

Comment:
Should we have some mlir output from these two examples with `commute_ppr` and `ppr_to_ppm`?